src/proto/FunctionRpc.proto

```diff
@@ -10,6 +10,7 @@ option go_package ="github.com/Azure/azure-functions-go-worker/internal/rpc";
 package AzureFunctionsRpcMessages;
 
 import "google/protobuf/duration.proto";
+import "deps/IdentityRpc.proto";
```

Comment:
Following the conventions similar to https://github.com/googleapis/googleapis/tree/master/google/bytestream, move new definition to its own folder

Reply:
Moved to `identity` folder.